pipeline.sh

```diff
@@ -9,8 +9,16 @@
 #SBATCH --export=NONE
 #SBATCH --get-user-env=L60
 
-SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-source "${SCRIPT_DIR}"/utils/header.sh
+# Retrieve original directory of submitted script.
+if [ -n "$SLURM_JOB_ID" ] ; then # If Slurm job.
+  SCRIPT_SUBMIT_DIR=$(scontrol show job "$SLURM_JOBID" | awk -F= '/Command=/{print $2}')
+  SCRIPT_SUBMIT_DIR="${SCRIPT_SUBMIT_DIR%% *}" # Removes anything starting at first space (so keep script path).
+else
+  SCRIPT_SUBMIT_DIR=$(realpath "$0")
+fi
+readonly SCRIPT_SUBMIT_DIR="${SCRIPT_SUBMIT_DIR%/*}" # Removes "/<scriptname>".
```

Comment:
Could be part of the if clause if you don't make the change mentioned in the previous comment.

Reply:
It was left outside the if clause as both cases have this part in common. One could use `dirname` instead and add it to both the if & else if less lines is preferred.